notification/build.gradle

```diff
@@ -24,3 +27,78 @@ dependencies {
     testImplementation "org.elasticsearch.test:framework:${es_version}"
     testImplementation "org.easymock:easymock:4.0.1"
 }
+
+shadowJar {
```

Comment:
this may not work when you have ISM and alerting both depending on notification. I would suggest to repackage at ES plugin level to avoid JarHell

Reply:
Tested and relocation/repackaging seems to happen during publish and not build. Alerting/ISM using notification should be fine for now, but we do need a long term solution for our collective plugins dependencies. Thanks for the help!